dace/frontend/python/parser.py

```diff
@@ -232,12 +254,13 @@ def infer_symbols_from_shapes(sdfg: SDFG, args: Dict[str, Any],
 class DaceProgram:
     """ A data-centric program object, obtained by decorating a function with
         `@dace.program`. """
-    def __init__(self, f, args, kwargs):
+    def __init__(self, f, args, kwargs, cli_args):
```

Comment:
why specifically in the python frontend?

Reply:
dito